tests/tests.js

```diff
@@ -180,7 +180,7 @@ var startTests = function (lscache) {
       var key = 'thekey';
       var value = 'thevalue';
       var minutes = 1;
-      lscache.set(key, value, minutes);
+      equal(lscache.set(key, value, minutes), true, 'We expect the value to be inserted successfully');
       setTimeout(function() {
         equal(lscache.get(key), null, 'We expect value to be null');
         start();
```

Comment:
There don't seem to be any tests that check for when it returns false, is it possible to add them, or is it too hard/not useful to mock the necessary environment for failures to happen?

Reply:
I thought I'd have time to address these today but unfortunately not. Please hang out for a bit longer, I'll try and get it done in the next week or so. Thanks.

Reply:
OK maybe that was a little bit Valve time but I got it done in the end. I've tested the scenarios which can reasonably be tested. I don't think we can test the other ones (disabled features, generic exceptions), unless you can think of a way.